pkg/resources/grant_privileges_to_account_role.go

```diff
@@ -246,7 +246,7 @@ var grantPrivilegesToAccountRoleSchema = map[string]*schema.Schema{
 					Description: "Configures the privilege to be granted on all objects in either a database or schema.",
```

Comment:
I guess this should say "on future". Not related to the pr, but yeah 😅.

Reply:
Changed.